estate/models/estate_property.py

```diff
@@ -0,0 +1,45 @@
+# -*- coding: utf-8 -*-
+
```

Comment:
```suggestion
```
The `# -*- coding: utf-8 -*-` line was necessary in Python 2 to specify the source file's character encoding because Python 2 defaulted to ASCII.

In Python 3, which Odoo now uses, UTF-8 is the default source code encoding. Therefore, this declaration is redundant and can be safely removed.